src/canvasObject.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var _ = require('underscore');
```

Comment:
@aeschylus Are we ok with having underscore be a dependency for the built iiifManifestLayouts module? I'm all for it; it'll definitely simplify things. Just making sure.